Code/src/main/java/nl/utwente/group10/ui/menu/FunctionMenu.java

```diff
@@ -114,8 +119,6 @@ protected void updateItem(FunctionEntry item, boolean empty) {
         graphBlockButton.setOnAction(event -> addBlock(new GraphBlock(parent)));
 
         // TODO remove once debugging is done
-        Button debugButton = new Button("Error All");
-        debugButton.setOnAction(event -> parent.errorAll());
```

Comment:
Are we really done with all the debugging? This code should really be the last line to go when we are done, doesn't make sense to throw away debugging tools before we finalise the code.